lua/nvim-tree/renderer/init.lua

```diff
@@ -50,6 +50,14 @@ local function _draw(bufnr, lines, hl_args, signs, extmarks)
       })
     end
   end
+
+  for line_nr, vlines in pairs(virtual_lines) do
+    vim.api.nvim_buf_set_extmark(bufnr, namespace_extmarks_id, line_nr, 0, {
```

Comment:
Usual paranoia - should we create another namespace for this?

Reply:
I'll make sure of that this week

Reply:
TY